examples/kubernetes/volume_scheduling/snapshot.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: snapshot.storage.k8s.io/v1alpha1
```

Comment:
Could you create a separate dir `snapshot` under `examples/kubernetes` with all the necessary manifest files just for this use case? I am trying to modularize the examples so that user can just pick whichever example they care about. It will be great if you also add some README.md under the snapshot dir with some instructions about how to execute the example.